app/views/courses/_votes.erb

```diff
@@ -0,0 +1,10 @@
+<p class="font-weight-bold text-uppercase mb-2">
+  <%= 'like'.pluralize.titleize %>
+</p>
+<p>
+<%= link_to course_vote_path(@course), class: 'voting-button', method: vote_method, remote: true do %>
+  <%= icon 'heart', variant: heart_fill %>
+<% end %>
+<%= @vote_total %>
+
```

Comment:
[nitpick] Indentation looks a bit off here.

Reply:
Actually ended up removing the partial.